adagrams/game.py

```diff
@@ -1,11 +1,120 @@
+import random 
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    available_letters = []
+    ten_random_letters = []
+
+    # I am iterating through each letter and number in the dictionary so that I can
+    # get a list (available_letters) showing all available letters. 
+    for letter, number in LETTER_POOL.items():
```

Comment:
[suggestion] Since you don't use `number`, you could switch to:

```python
    for letter in LETTER_POOL:
```

which iterates over all the keys of the dictionary.

This also means you could create `available_letters` without a loop at all:

```python
    available_letters = list(LETTER_POOL)
```

...or if you want it to be more explicitly that you're getting the keys...

```python
    available_letters = list(LETTER_POOL.keys())
```

It's good that you got practice looping over `LETTER_POOL.items()` though!

Reply:
HOWEVER, looking at the `while` loop below, I think you *do* need to use `number`! Your `available_letters` list should have a copy of each letter, for it to work the way you intend it to. See if you can figure out how to modify this loop to fix the issue I identified.